C0T0/Classes/ApiDataSource/URLRequestBuilder.swift

```diff
@@ -10,20 +10,41 @@ protocol URLRequestBuilder {
 }
 
 class StandardURLRequestBuilder: URLRequestBuilder {
-
+    private enum Constants {
+        static let http = "http"
+        static let https = "https"
+        static let slashes = "://"
+        static let httpProtocol = Constants.http + Constants.slashes
+        static let httpsProtocol = Constants.https + Constants.slashes
+    }
+    
     private let host: String
-
+    private let scheme: String
     init(host: String) {
+        var host = host
+        var scheme = ""
+        if host.contains(Constants.httpProtocol) {
+            host = host.replacingOccurrences(of:Constants.httpProtocol , with: "")
```

Comment:
You can just make an assignment like: `self.host = host.replacingOccurrences(of:Constants.httpProtocol , with: "")`. I will allow you to remove two local variables and four lines of code :)

Reply:
Btw, you missed space after `:` :)